{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/services.py

```diff
@@ -22,3 +22,13 @@ def create_user_account(email, password, first_name="", last_name=""):
 
 def get_user_by_email(email: str):
     return get_user_model().objects.filter(email__iexact=email).first()
+
+
+{%- if cookiecutter.add_graphql == "y" %}
+def get_user_by_id(user_id):
+    user_model = get_user_model()
+    try:
+        return user_model.objects.get(id=user_id)
+    except user_model.DoesNotExist:
+        return None
```

Comment:
```suggestion
def get_user_by_id(user_id):
    return get_user_model().objects.filter(id=user_id).first()
```

Also, I'm not sure if we really want to wrap them up with graphql. This is a generic service, so we can just leave it there.